pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -616,9 +618,29 @@ func (lr *LegacyRegistry) ProcessPriorityPolicy(policy config.PriorityPolicy, co
 	}
 
 	if policy.Argument.RequestedToCapacityRatioArguments != nil {
-		configProducerArgs.RequestedToCapacityRatioArgs = &noderesources.RequestedToCapacityRatioArgs{
-			RequestedToCapacityRatioArguments: *policy.Argument.RequestedToCapacityRatioArguments,
+		policyArgs := policy.Argument.RequestedToCapacityRatioArguments
```

Comment:
This should temporary. Add a comment just in case.

Reply:
I'm not sure if a comment would reduce confusion. If it does, it should be also added to other places in this - and potentially other - files. What would you say here?

Reply:
You are right. Instead add a note in the PR description saying that we are using external types temporarily and that there will be a follow up PR.